src/test/java/org/jabref/logic/importer/fetcher/ZbMATHTest.java

```diff
@@ -44,18 +43,42 @@ void setUp() throws Exception {
         donaldsonEntry.setField(StandardField.TITLE, "An application of gauge theory to four dimensional topology");
         donaldsonEntry.setField(StandardField.VOLUME, "18");
         donaldsonEntry.setField(StandardField.YEAR, "1983");
-        donaldsonEntry.setField(new UnknownField("zbl"), "0507.57010");
+        donaldsonEntry.setField(StandardField.ZBL_NUMBER, "0507.57010");
     }
 
     @Test
     void searchByQueryFindsEntry() throws Exception {
         List<BibEntry> fetchedEntries = fetcher.performSearch("an:0507.57010");
-        assertEquals(Collections.singletonList(donaldsonEntry), fetchedEntries);
+        assertEquals(donaldsonEntry, fetchedEntries.get(0));
```

Comment:
Why did you change this? The search should only return one entry, or not?

Reply:
I had some problems with the tests, and changed that because I thought this was the cause but it was not. I will change it back to assertEquals(Collections.singletonList(donaldsonEntry), fetchedEntries) and in the other tests, too.